lib/shoryuken.rb

```diff
@@ -26,7 +27,8 @@ module Shoryuken
     queues: [],
     aws: {},
     delay: 0,
-    timeout: 8
+    timeout: 8,
+    polling_strategy: Polling::WeightedRoundRobin,
```

Comment:
:+1: All for extracting this kind of stuff out. It makes things more extensible. However we probably want a more concrete interface here. I would expect this to be a `PollingStrategy::WeightedRoundRobin`. I would also expect a `PollingStrategy::Base` that I could inherit from that enforces a certain interface if I wanted to write my own.